projects/github.com/cosmtrek/air/package.yml

```diff
@@ -0,0 +1,74 @@
+# this is a basic template for packages we strongly recommend looking at the
+# contents of both:
+#
+# * https://github.com/pkgxdev/pantry
+#
+# as a learning guide and reference. For example, if you are package a `go`
+# project, then browse the listings and find a go package.
+
+
+# this is the source tarball. Swap the version out for {{version}} or we won’t
+# be able to upgrade the package when it updates!
+distributable:
+  url: https://github.com/cosmtrek/air/archive/v{{version}}.tar.gz
+  # this obscure option dictates how many components are removed from the
+  # extracted tarball. It only works with tarballs. You almost always want
+  # `1`. If after extraction the src-directory is funky try `0` or `2`.
+  strip-components: 1
+  # alternatively, we can use direct git checkouts from refs
+  # but we should prefer official source distributions.
+  # often CI/CD will do some post-processing of the source
+  # for release, including setting version numbers, running
+  # autotools, etc.
+  # url: git+https://github.com/user/repo
+  # ref: v{{version}}}
+
+# if there’s a github then we can parse the versions
+versions:
+  github: cosmtrek/air
+
+platforms:
+  - darwin
+  - linux/x86-64
```

Comment:
does it not build on linux+aarch64? please add a FIXME comment with the issue if so for future reference.